apps/dapp/src/components/Pages/Core/DappPages/Dashboard/DashboardContent.tsx

```diff
@@ -95,9 +95,9 @@ const DashboardContent = ({ selectedDashboard = DashboardType.TREASURY_RESERVES_
       <Header>
         <HeaderTextContainer>
           <HeaderTitle>{dashboard.title}</HeaderTitle>
-          <LinkIcon onClick={() => window.open(dashboard.link, "_blank")}/>
+          <LinkIcon onClick={() => window.open(dashboard.link, '_blank')} />
         </HeaderTextContainer>
-        <HeaderText>{dashboard.description}</HeaderText>
+        <HeaderText dangerouslySetInnerHTML={{ __html: dashboard.description }} />
```

Comment:
Because we have a link in the description (up until this point, it was only text).